app/config/collections.php

```diff
@@ -3595,18 +4277,32 @@
         ],
         'indexes' => [
             [
-                '$id' => '_key_function',
+                '$id' => '_key_resourceInternalId',
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['resourceInternalId'],
+                'lengths' => [Database::LENGTH_KEY],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => '_key_resourceId',
                 'type' => Database::INDEX_KEY,
-                'attributes' => ['functionInternalId'],
+                'attributes' => ['resourceId'],
                 'lengths' => [Database::LENGTH_KEY],
                 'orders' => [Database::ORDER_ASC],
             ],
+            [
+                '$id' => '_key_resourceType',
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['resourceType'],
+                'lengths' => [100],
```

Comment:
Can be left empty , for an attribute length with same length.

Reply:
left empty